erpnext/stock/get_item_details.py

```diff
@@ -508,7 +514,8 @@ def _get_item_tax_template(args, taxes, out=None, for_validate=False):
 		return None
 
 	# do not change if already a valid template
-	if args.get('item_tax_template') in taxes:
+	if args.get('item_tax_template') in [t.item_tax_template for t in taxes]:
```

Comment:
Checks for memberships are better done over sets instead. 
```suggestion
	if args.get('item_tax_template') in {t.item_tax_template for t in taxes}:
```

ref: https://stackoverflow.com/a/2831242/10309266